mdata/notifierKafka/cfg.go

```diff
@@ -20,6 +21,9 @@ var Config *sarama.Config
 var OffsetDuration time.Duration
 var OffsetCommitInterval time.Duration
```

Comment:
with the exception of Enabled, all these variables don't need to exported anymore because we only use them in the package itself. in fact we probably also don't need the two separate files anymore. everything that's in cfg.go can be moved into notifierKafka.go (although you can also keep it separate if you want, i don't care much)